### PR TITLE
replace `rlang::warn()` with `cli::cli_warn()`

### PR DESCRIPTION
**Pull request by EmilHvitfeldt**
ref: https://github.com/tidymodels/dials/issues/311

Nothing super special here

### CODE REVIEW

File: R/grids.R [hunk 1]
```diff
@@ -49,7 +49,10 @@
 grid_regular <- function(x, ..., levels = 3, original = TRUE, filter = NULL) {
   dots <- list(...)
   if (any(names(dots) == "size")) {
-    rlang::warn("`size` is not an argument to `grid_regular()`. Did you mean `levels`?")
+    cli::cli_warn(
+      "{.arg size} is not an argument to {.fn grid_regular}.
+      Did you mean {.arg levels}?"
```
Comment (hfrick) [inline R/grids.R:54]: could you turn the second sentence into an info bullet via `"i" = "Did you mean.."`?

**Review by hfrick — COMMENTED** [review]
Nice! One small ask in the comments ℹ️

File: R/grids.R (hunk 1 above)
Comment (EmilHvitfeldt) [reply R/grids.R]: done in https://github.com/tidymodels/dials/pull/350/commits/678cbe5c0d8471358e6b66fc28269a74458b095a

**Comment by hfrick** [comment]
Ah, I meant for all of them, to keep them in sync! Done it now, merge time! ✅

**Pull request merged**